report/4.Solution/DesignOfCompiler.tex

```diff
@@ -0,0 +1,26 @@
+% !TeX root = ../main.tex
+\section{Design of Compiler}
+The compiler is composed of three primary parts, which is explained in the following sections.
+The explanation both covers code snippets of the compiler as well as reasoning behind development decisions.
```

Comment:
Both covers -> covers both
as well as reasoning -> as well as the reasoning (måske også et comma før denne (, as well), kun måske)